### ISSUE TITLE
DetailsList row is not highlighted on mouse hover in high contrast mode.

### ISSUE THREAD

**Issue by Harkesh030679** [microsoft/fluentui#13177]
Grid row is not highlighted on mouse hover in high contrast mode using DetailsList - Custom Item Columns
### Environment Information
OS: Win 10
Browser: Edge

### Describe the issue:

Grid row is not highlighted on mouse hover in high contrast mode using 'DetailsList - Custom Item Columns' controls

### Please provide a reproduction of the issue in a codepen:
Please refer below link to repro the issue. Please change settings of system to high contrast
https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/customitemcolumns
Attached snap

![GridRow_HC](https://user-images.githubusercontent.com/62233836/82037448-c4080000-96bf-11ea-9cd6-c841211bdf79.png)

#### Actual behavior:

Grid row is not highlighted on mouse hover in high contrast mode using 'DetailsList - Custom Item Columns' controls

#### Expected behavior:

Grid row should be highlighted on mouse hover in high contrast mode using any relevant color code.

**Comment by ecraig12345** [comment]
Thanks for reporting this, but I'm not sure if it's something we're going to change (will have to discuss with the team). There is still some indication of which item is hovered since the checkbox is shown. I'm also a bit concerned that any change would be unexpected by our various partners using DetailsList.

I think at most, the rows should be outlined on hover like Windows File Explorer does, rather than having a background color. But I'll have to talk with some people to figure out whether to do that.
![image](https://user-images.githubusercontent.com/5864305/82082329-b0976d80-969c-11ea-8d6e-641a968dc105.png)

**Comment by ecraig12345** [comment]
@betrue-final-final What do you think we should do here?

**Comment by betrue-final-final** [comment]
Ok, there's a common pattern where selection and hover use the same visuals, which is Highlight background and HighlightText foreground. The outline is generally a focus, but can be also used for hover. I would either do nothing or use the common selection visuals.

**Comment by ecraig12345** [comment]
After further discussion with @betrue-final-final offline, we're going to fix this. Will hopefully be pretty straightforward since it's copying the styling used for the existing selection state.

**Comment by SalmanSaberi** [comment]
Hi Team
Please let us gives us an ETA by when this issue can be closed ?
